test/e2e/apimachinery/aggregator.go

```diff
@@ -550,6 +576,368 @@ func TestSampleAPIServer(f *framework.Framework, aggrclient *aggregatorclient.Cl
 	cleanTest(client, aggrclient, namespace)
 }
 
+// TestAPIService uses the sample-apiserver
+// (v1beta1.wardle.example.com) to test replace
+// status and deleteCollection APIService endpoints
+func TestAPIService(f *framework.Framework, aggrclient *aggregatorclient.Clientset, image string) {
```

Comment:
TestSampleAPIServer already exercises routing requests to the aggregated server... if the intent of this is to exercise CRUD on the `apiservices` and `apiservices/status` endpoints, can that be done more simply?

Reply:
From the last conformance meeting when @smarterclayton looked over this test the feedback was that the structure looked okay and that there where only some minor style issues to address.

I used the initial setup of the original test as a template to test these two endpoints as I thought this would be the minimum required resources for testing these two endpoints. I would need some guidance on how much of the test setup can be paired back.

Reply:
cc @smarterclayton ... what is the new test intended to exercise? the routing is already exercised by the existing TestSampleAPIServer test